### PR TITLE
[feat][doc] add configurations for broker load balancing

### PR DESCRIPTION
**Pull request by Anonymitaet**
## What change does this PR introduce?

This PR adds configurations for broker load balancing (topics in the red box below)

image

## Doc development plan - Read before review

For the current doc of Broker Load Balancer, we want to tackle the following challenges:
- Create new documentation to cover all aspects of the feature (especially for the new extensible load balancer).
- Fill in any content gaps that may exist.
- Consolidate existing documentation into the new IA, as it is currently scattered and disorganized.
- Organize the documentation in a cohesive manner, ensuring it flows logically.

Consequently, we’ve created the following content development plans:
- [Broker Load Balancing | Information Architecture](https://xmind.works/share/dzbyKsnS)  
  This document presents a high-level architecture overview, including how we address content gaps and reconcile existing documentation.

- [Broker Load Balancing | Doc Outline](https://docs.google.com/document/d/1zshyn93XJp8Y-uvYLTnFCrcUWaHxeiTjPqdaUrR6zf0/edit#heading=h.6iz0wf853c5u)
  This document delves into the specifics of the content development plan, providing purpose and context for each section.

- [x] `doc` 

## Ackowledgements

@Demogorgon314 thanks for your technical guidance and great support!

### CODE REVIEW

File: static/reference/next/config/README.md [hunk 1]
```diff
@@ -32,3 +32,72 @@ You can set specific configurations through Java properties.
 | **Property**       | **Description**                                              |
 | ------------------ | ------------------------------------------------------------ |
 | pulsar.enableUring | Use `io_uring` instead of `epoll` as network IO mode. `-Dpulsar.enableUring=1` means this feature is enabled. |
+
+## Broker load balancing configurations
```
Comment (Anonymitaet) [inline static/reference/next/config/README.md:36]: For reviewers:

The detailed descriptions of broker load balancing configurations are shown at https://pulsar.apache.org/reference/#/next/config/reference-configuration-broker. This page is generated from code files automatically. All configurations including those related to broker load balancing are listed on that page without appropriate sorting. 

To give users a clear and quick understanding of broker load balancing configurations, I've added this brief summary.

**Comment by Anonymitaet** [comment]
@Demogorgon314 @heesung-sn @D-2-Ed @momo-jun @Huanli-Meng could you please help review this PR?
Thank you all!

**Review by momo-jun — APPROVED** [review]
LGTM

File: static/reference/next/config/README.md [hunk 2]
```diff
@@ -32,3 +32,72 @@ You can set specific configurations through Java properties.
...
+- loadBalancerReportUpdateThresholdPercentage
+- loadBalancerReportUpdateMaxIntervalMinutes
+- loadBalancerBandwithInResourceWeight
+- loadBalancerBandwithOutResourceWeight
+- loadBalancerCPUResourceWeight
+- loadBalancerMemoryResourceWeight
+- loadBalancerDirectMemoryResourceWeight
+- loadBalancerHistoryResourcePercentage
+  
+### TopK bundle load data
```
Comment (Demogorgon314) [inline static/reference/next/config/README.md:57]: I think this title is `Bundle load data`

**Comment by heesung-sn** [comment]
I think we can move this content inside Admin->LoadBalance page.

**Comment by Anonymitaet** [comment]
> I think we can move these configs when we explain each component's operation.

@heesung-sn I'll add the link of this conf page to https://pulsar.apache.org/docs/next/concepts-broker-load-balancing-types/

**Pull request merged**